backend/src/dao/TeamEventsDao.ts

```diff
@@ -60,6 +60,6 @@ export default class TeamEventsDao {
     const docRefs = await teamEventsCollection
       .select('name', 'date', 'numCredits', 'hasHours', 'uuid')
       .get();
-    return docRefs.docs.map((doc) => doc.data() as TeamEventInfo);
+    return Promise.all(docRefs.docs.map(async (doc) => doc.data() as TeamEventInfo));
```

Comment:
I don't believe `doc.data()` returns a promise. 
![image](https://user-images.githubusercontent.com/65922473/234986557-1015889c-6391-4935-96b1-4f9bb7c99aed.png)

Reply:
I thought this change had fixed an error I was getting, but it seems like both ways work. I will change it back.